nilearn/input_data/multi_nifti_masker.py

```diff
@@ -211,6 +214,67 @@ def fit(self, imgs=None, y=None):
         self.mask_img_.get_data()
         return self
 
+    def transform_imgs(self, imgs_list, confounds=None, copy=True, n_jobs=1):
+        ''' Prepare multi subject data in parallel
```

Comment:
We tend to use triple double quotes reasonably consistently everywhere else for docstrings.